sentinelhub/config.py

```diff
@@ -58,7 +58,8 @@ class _SHConfig:
             ('max_opensearch_records_per_query', 500),
             ('max_download_attempts', 4),
             ('download_sleep_time', 5),
-            ('download_timeout_seconds', 120)
+            ('download_timeout_seconds', 120),
+            ('number_of_download_processes', 1)
```

Comment:
Could you also add the parameter description in `SHConfig` class docstring? I imagine this parameter could easily be misinterpreted if we don't explain what it means and what it is used for.

Reply:
Done.